packages/wrangler/src/config/validation.ts

```diff
@@ -290,11 +306,18 @@ function normalizeAndValidateBuild(
       // change in the future, so we do a check anyway
       command
         ? configPath
```

Comment:
at this point, maybe it's just better if this is just
```suggestion
      command && configPath
```